docs/operation/troubleshooting.md

```diff
@@ -0,0 +1,79 @@
+---
+id: troubleshooting
+title: Troubleshooting
+sidebar_label: Troubleshooting
+---
+
+This section contains information to help you identify and resolve errors you may encounter.
+
+### Known errors and issues
+
+#### lookup failed for scheduledEventID to activityID
+
+See [Nondeterministic troubleshooting steps](#nondeterministic-troubleshooting-steps)
+
+#### unable to find activityID for the scheduledEventID
+
+See [Nondeterministic troubleshooting steps](#nondeterministic-troubleshooting-steps)
+
+#### nondeterministic Workflow error
+
+See [Nondeterministic troubleshooting steps](#nondeterministic-troubleshooting-steps)
+
+#### Potential deadlock
+
+The Temporal Platform requires that a Workflow Function Execution run within 1 second, or it is considered _deadlocked_.
+
+Use the Workflow replay APIs to replay the Workflow Execution from its Event History to try and replicate the issue.
+
+- [How to replay a Workflow Execution in Go](/docs/go/how-to-replay-a-workflow-execution-in-go)
+
+#### ApplicationFailure: 'MyFunction' is not a function
+
+Verify that the Worker Entities have the proper Workflow Types, Activity Types registered and are listening to the proper Task Queues.
+
+#### Workflow did not register a handler for MyQuery
+
+Verify that the Worker Entities have the proper Workflow Types, Activity Types registered and are listening to the proper Task Queues.
+
+#### [ERROR] Module not found: Error: Can't resolve
```

Comment:
Is this a TypeScript-specific error? I don't think these language specific error messages need to be in language specific pages.